budget/tests.py

```diff
@@ -758,7 +759,7 @@ def api_add_member(self, project, name, weight=1):
     def get_auth(self, username, password=None):
         password = password or username
         base64string = base64.encodestring(
-                '%s:%s' % (username, password)).replace('\n', '')
+            ('%s:%s' % (username, password)).encode()).decode().replace('\n', '')
```

Comment:
This encode() decode() doesn't looks right, it should be fixed once you import ``unicode_literals``